app/models/manageiq/providers/physical_infra_manager/vm.rb

```diff
@@ -0,0 +1,7 @@
+class ManageIQ::Providers::PhysicalInfraManager::Vm < ::Vm
+  default_value_for :cloud, false
+
+  def self.display_name(number = 1)
+    n_('Virtual Machine', 'Virtual Machines', number)
+  end
```

Comment:
Is this not inherited?

Reply:
The base class is `n_('VM and Instance', 'VMs and Instances', number)`

Reply:
ohhhh interesting